documentation_website/how_to_use.qmd

```diff
@@ -0,0 +1,147 @@
+---
+title: ""
+format: html
+---
+
+## Installing BON in a Box
+
+The instructions for how to install BON in a Box on your computer can be found [here](how_to_install.qmd).
+
+## Connecting to the server and running BON in a Box
+
+1.  Connect to the server by running `./server-up.sh` in a Linux terminal (e.g. Git Bash or WSL on Windows or Terminal on Mac). Make sure you are in the bon-in-a-box-pipeline directory on your computer.
+2.  Open a web browser and navigate to <http://localhost/>.
+
+# User Guide
+
+Here is a detailed guide on how to run BON in a Box locally on your computer. This guide outlines the features of BON in a Box and how to use them.
+
+## Home
+
+Under the `Home` tab, you will find a description of the BON in a Box modeling tool, as well as all of its featured pipelines. And your most recent pipeline runs.
+
+::: {style="margin-top: 40px;"}
+:::
+
+![](images/Image7.png)
+
+::: {style="margin-top: 40px;"}
+:::
+
+## Run a script
+
+The `Run a script` tab lets you run an individual script. Scripts are individual parts of pipelines designed to perform one well-defined task at a time, like extracting data from a file, cleaning files, or applying a specific formulas.
+
+::: {style="margin-top: 40px;"}
+:::
+
+![](images/Image8.png)
+
+::: {style="margin-top: 40px;"}
+:::
+
+1.  Select from the drop down menu the script you would like to run.
+
+    -   Under the menu you have a description of the script, with its authors and references.
+
+2.  Fill out the form with the input parameters the script should use.
+
+    -   Scripts will run with default parameters if no inputs are specified. Sometimes scripts require specific files that are pulled from other parts of a pipeline, so you may not be able to run the script by itself with the default parameters. However, all pipelines should run successfully with the default parameters.
+
+    -   Some inputs can simply be typed in, while others are options.
+
+    -   Each input field includes a description and its purpose on the right of the input box. Here you will also see the *type* of the input.
+
+    -   The form might ask you for a file. In order to provide a file that you have locally on your computer, upload or copy it to the `userdata` folder. You can then refer to it with a url as such: `/userdata/myFile.shp`, or `/userdata/myFolder/myFile.shp` if there are subfolders.
+
+    -   **Please ensure each input matches the required type that is specified in the input description.** **For any details on input types, please refer to Step 3 of [this page](how_to_contribute.qmd).**
+
+    -   The `Input yaml` tab can be used to see how your inputs are being passed in to the script. By scrolling down, you can also see how the inputs were defined by the author(s).
```

Comment:
This can be useful to send your exact run parameters to a co-worker or the BON in a Box team.